src/metaTypes.h

```diff
@@ -219,6 +219,12 @@ typedef struct
                                   //   meta data
    } MET_FieldRecordType;
 
+#if __cplusplus >= 201103L
```

Comment:
Is suspect this will not work with visual studio. Does it matter ?

Usually compiler compatibility by inspecting __cplusplus, but MSVC doesn't set __cplusplus.
See https://blogs.msdn.microsoft.com/vcblog/2016/06/07/standards-version-switches-in-the-compiler/.
Microsoft: "We won’t update __cplusplus until the compiler fully conforms to
the standard. Until then, you can check the value of _MSVC_LANG."

Reply:
I don't think it matters because I don't remember there being a "you're missing `override`" warning in MSVC anyways.

Reply:
Good point. This also corroborate with my observation in an other project.